dkron/management/commands/run_dkron.py

```diff
@@ -55,7 +55,6 @@ def download_dkron(self):
         return str(exe_path), str(bin_dir)
 
     def handle(self, *_, **options):
-        # TODO: check if there's any shutdown we should care before execv()
```

Comment:
Should we move this to a new issue?

Reply:
I had that TODO as a generic comment of "is it ok to execv in a django command?" as there could be things/cleanup that should be performed.

But I'm not sure how to even start to check that so I was just plainly removing it 😄

Reply:
But we could track it in an issue actually, it doesn't hurt!

Reply:
https://github.com/surface-security/django-dkron/issues/32 created.

I'm not planning on hitting other smells for the time being, so if you're ok with this PR @gsilvapt, it's ready to merge

Reply:
Thanks! This will help fix some of those smells and we can continue afterwards.